graphrag/query/__main__.py

```diff
@@ -11,6 +11,7 @@
 from .cli import run_global_search, run_local_search
 
 INVALID_METHOD_ERROR = "Invalid method"
+DYNAMIC_SELECTION_ERROR = "Dynamic community selection only supports global search"
```

Comment:
If that's the case, should we even select a method when doing dynamic selection?